docs/organizations/managing-security-and-risk.md

```diff
@@ -26,9 +26,13 @@ Each panel shows the total count of matching items and contains a **Review** but
 When viewing the dashboard, you can:
 
 -   Limit the total counts in each panel to a specific set of severities or repositories by clicking the **Severity** or **Repository** drop-downs above the main area.
--   Export a list of items as a CSV file by clicking the **Export CSV** button in the top right-hand corner of the page. The exported list always includes all items, ignoring any applied filters.
+
 -   Review the [severity assignment rules](#item-severities-and-deadlines) by clicking the **See rules** button in the top right-hand corner of the page.
 
+-   Export a list of items as a CSV file by clicking the **Export CSV** button in the top right-hand corner of the page. The exported list always includes all items, ignoring any applied filters.
+
+    !!! info "The **Export CSV** feature is available only to organization admins and organization managers"
```

Comment:
@claudiacarpinteiro Should we also mention elsewhere on the page that each user only sees items relating to the repos they can access? Thoughts on the ideal placement for this info? Since it impacts the entirety of the feature, should we mention it in the intro section?

Reply:
I looked for something similar, for example, the [Organization Overview](https://docs.codacy.com/organizations/organization-overview/) page, where we have in the introduction:
"The Organization Overview provides an overview of the **repositories belonging to your Git provider organization that you [follow on Codacy](https://docs.codacy.com/organizations/managing-repositories/)**."

We could have something similar in [this introduction](https://github.com/codacy/docs/pull/1965/files#diff-30a735ab496ba1650575435a2195b32444487dbb269bb4b685b2e1ad8c7ebe53R3):
"The Security and risk management feature helps you quickly identify, track, and address security issues by automatically opening time-bound, prioritized action items whenever Codacy detects security issues in the **organization repositories that you follow on Codacy** or in your connected Jira instance."

Reply:
I'm not convinced. SRM lets you *see* the items for repos you follow, but it tracks all of them, org-level. And admins/org managers can see all of them anyway, so this sentence wouldn't apply to them.

I opted to update the roles and permissions table to account for this, leaving the intro unchanged.